app/views/posts/index.html.erb

```diff
@@ -1,5 +1,27 @@
-<h1>All posts for User #<%= @user %></h1>
+<%= render partial: "shared/user", locals: {user: @user } %>
 
-<% @posts.each do |post| %>
-    <h3>Id: <%= post.id %>, Title: <%= post.title %>, Text: <%= post.text %>, Created at: <%= post.created_at %></h3>
+<% if false %>
+    <% @posts.each do |post| %>
+        <h3>Id: <%= post.id %>, Title: <%= post.title %>, Text: <%= post.text %>, Created at: <%= post.created_at %></h3>
+    <% end %>
 <% end %>
+
+<% @posts.each_with_index do |post, index| %>
+    <%= render partial: "shared/post", locals: {post: post, index: index } %>  
+
+    <div class='div-comments'>
+    <% post.five_recent_comments(post.id).each_with_index do |comment, index| %>
+       <div class='comments'>
+          <span style="font-weight :bold font-size: 10px">Username <%= comment.user_id %>:</span> <%= comment.text %>
+        </div>
+    <% end %>        
+    </div>
+          
+<% end %>
```

Comment:
- [ ] Kindly ensure the design of your posts page matches the wireframe given, and check the positioning of comments and likes:

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/204990424-e0cfed48-10ca-4ac9-98b6-fc6efa822f74.png) |  ![image](https://user-images.githubusercontent.com/61408860/204990475-b27caefc-0f49-44d8-a53d-e0bc5e834581.png)

Reply:
@danmainah Done !